src/features/architect/tradeMachine/TradeTeamCard.jsx

```
@@ -300,6 +304,15 @@ const TradeTeamCard = ({
               <span className="font-semibold text-white/80">
                 {formatSalary(allowableIncomingNoTPE)}
               </span>
+              {salaryMatchingResult?.ruleLabel && (
+                <span 
+                  className="ml-1 text-white/40" 
+                  title={salaryMatchingResult.formulaUsed}
+                  aria-label={`Rule: ${salaryMatchingResult.ruleLabel}. Formula: ${salaryMatchingResult.formulaUsed}`}
+                >
+                  ({salaryMatchingResult.ruleLabel})
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)